packages/ui/src/common/types/helpers.ts

```diff
@@ -1,3 +1 @@
 export type Reducer<Accumulator, Value = Accumulator> = (acc: Accumulator, value: Value, index: number) => Accumulator
-
-export type MakeTransient<T, K extends keyof T & string> = Omit<T, K> & { [P in `$${K}`]: T[K] }
```

Comment:
@jodator I can't figure out why [prettier fails on the template literal types](https://github.com/Joystream/pioneer/runs/3125801868) since [the current version is supposed to have typescript 4.3 support](https://github.com/prettier/prettier/blob/main/CHANGELOG.md#231), any idea ?

Reply:
Hmm, hard to tell I didn't have that problem.